# Robot:
# collision_box: [1.0, 1.0, 0.6]
collision_box: [1.3, 1.2, 0.0] #[1.0,0.7,0.0]


# Environment:
env_box: [-100, 100, -100, 100, 0.5, 0.5] # xmin, xmax, ymin, ymax, zmin, zmax


# Planner parameters:
timeout: 8.0 #5s
incremental_distance: 0.2 #0.3m
connect_goal_ratio: 0.2
goal_reach_distance: 0.4 #m
map_resolution: 0.1 # this should be at least bigger or equal to the true map resolution
ignore_unknown: true
max_shortcut_dist: 5.0

# Planner Visualization:
vis_RRT: False # does not support for RRT*
vis_path: True

# RRT* parameter:
neighborhood_radius: 1.0 #m
max_num_neighbors: 10